libs/wire-api/test/unit/Test/Wire/API/User/Identity/AuthId.hs

```diff
@@ -24,40 +24,6 @@ specs = do
     prop "arbitrary AuthId values are not missing in validStatesExpected" $ \(authId :: AuthId) -> do
       allSamples `shouldContain` [Equiv authId]
 
-  xdescribe "what's an email" $ do
```

Comment:
I removed this only because i'd like to run all other tests on CI first.
- [ ] rever this

Reply:
Reverted in separate PR #1437